base_export_manager/models/res_users.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+###############################################################################
```

Comment:
Please use the new short OCA header - https://github.com/OCA/maintainer-tools/blob/master/template/module/exceptions.py#L3